.github/workflows/block-merge-eol.yml

```
@@ -27,4 +27,10 @@ jobs:
  
       - name: Checking if ${{ env.server_major }} is EOL
         run: |
-          php -r 'echo json_encode(require_once "config.php");' | jq --arg version "${{ env.server_major }}" '.stable[$version]["100"].eol' | grep --silent -i 'false'
+          if php -r 'echo json_encode(require_once "config.php");' | jq --arg version "${{ env.server_major }}" '.stable[$version]["100"].eol' | grep --silent -i 'false'; then
+            exit 0
+          fi
+
+          if ! php -r 'echo json_encode(require_once "config.php");' | jq --arg version "${{ env.server_major }}" '.beta[$version]["100"].eol' | grep --silent -i 'false'; then
+            exit 1
+          fi
```

You can use fallback in jq: https://stedolan.github.io/jq/manual/#Alternativeoperator://
```suggestion
          if php -r 'echo json_encode(require_once "config.php");' | jq --arg version "${{ env.server_major }}" '.stable[$version]["100"].eol // .beta[$version]["100"].eol'
' | grep --silent -i 'false'; then
            exit 0
          fi
```